Modules/zlibmodule.c

```diff
@@ -1436,8 +1436,6 @@ static PyObject *
 zlib_crc32_impl(PyObject *module, Py_buffer *data, unsigned int value)
 /*[clinic end generated code: output=63499fa20af7ea25 input=26c3ed430fa00b4c]*/
 {
-    int signed_val;
-
```

Comment:
Now the code is the same as `zlib_adler32_impl()`, here remove an unnecessary var `signed_val `.